opencti-platform/opencti-front/src/private/components/workspaces/dashboards/Dashboard.jsx

```diff
@@ -140,7 +133,28 @@ const DashboardComponent = ({ workspace, noToolbar }) => {
     }
     saveManifest(newManifest);
   };
+  const getMaxY = () => {
+    const maxY = Object.values(manifest.widgets).reduce(
+      (max, n) => (n.layout.y > max ? n.layout.y : max),
+      0,
+    );
+    return maxY;
```

Comment:
```suggestion
    return Object.values(manifest.widgets).reduce(
      (max, n) => (n.layout.y > max ? n.layout.y : max),
      0,
    );
```

Eslint was not happy